frontend/src/components/sidebar/sideBar.tsx

```diff
@@ -0,0 +1,27 @@
+import './sideBar.css'
+import { Link } from 'react-router-dom';
+
+
+
+
+const Sidebar = () => {
+	
+	const userData = JSON.parse(localStorage.getItem("auth") || "{}");
```

Comment:
why unnecessary tab space over here? please configure prettier on your system to avoid linting related issues.